packages/components/src/templates/drawer.stories.ts

```diff
@@ -54,29 +54,111 @@ export const Default = {
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Distinction
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Our Management
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+          </sd-navigation-item>
+          <sd-navigation-item vertical
+            >Union Investment Group
+            <sd-navigation-item vertical indented slot="children" current>
+              Investor protection
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Distinction
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Our Management
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+          </sd-navigation-item>
+        </nav>
+      </div>
     </sd-drawer>
+    <script>
+      function handleSecondLevel() {
+        document.querySelector('.level-one').classList.toggle('hidden');
+        document.querySelector('.levelTwo').classList.toggle('hidden');
+        document.querySelector('#returnButton').classList.toggle('hidden');
+      }
+    </script>
   `
```

Comment:
Looking at the figma file, it still is missing some details either in figma or here:
- in Figma, the description is missing
- in Storybook, the mobile screens are missing

Reply:
Hey @karlbaumhauer,

- Description was unecessary so I removed it from code
- The mobile screen in storybook is not implement seperatly but the component it self is responsive. Therefore just use the dev tools to see a mobile version:
![image](https://github.com/user-attachments/assets/85c111fc-08f8-4743-a9b4-6c940d0033eb)